pkg/k8sutil/metadata.go

```diff
@@ -80,6 +80,20 @@ func SetNodeAnnotations(nc v1core.NodeInterface, node string, m map[string]strin
 	})
 }
 
+// SetNodeAnnotationsLabels sets all keys in a and l to their respective values in
+// node's annotations and labels, likewise
+func SetNodeAnnotationsLabels(nc v1core.NodeInterface, node string, a, l map[string]string) error {
+	return UpdateNodeRetry(nc, node, func(n *v1api.Node) {
```

Comment:
Or just call the SetNodeLabels and SetNodeAnnotation methods that are already there.

Reply:
Is there any value in doing it in a single transaction?